inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/AbstractInfoPanel.java

```diff
@@ -171,7 +171,7 @@ public ViewMode(String id, CompoundPropertyModel<? extends KBObject> compoundMod
             Label identifier = new Label("idtext"); 
             TooltipBehavior tip = new TooltipBehavior();
             tip.setOption("autoHide", false);
-            tip.setOption("content",
+            tip.setOption(CONTENT_MARKUP_ID,
```

Comment:
The `CONTENT_MARKUP_ID` is used to identify a Wicket component in the HTML code. However, the "content" literal replaced here does refer to the option `content` that is set on the tooltip - the two have nothing to do with each other. The constant `CONTENT_MARKUP_ID` cannot be used here.

Reply:
@JaviValera if you address reckart's comment above, the other two changes can be merged.